src/chapters/15-part-2-recap.md

```diff
@@ -0,0 +1,39 @@
+Congratulations, you finished the second part of the tutorial!
+
+There was a lot of concent to cover in part two. To recap, here is what you learned:
+
+<h3><a href="../11-route-params/">Chapter 11</a></h3>
+
+* Routes with dynamic segments
+* Links with dynamic segments
+* Component tests with access to the router
+* Accessing parameters from dynamic segments
+* Sharing common setup code between tests
+
+<h3><a href="../12-service-injection/">Chapter 12</a></h3>
+
+* Splattributes and the `class` attribute
+* The router service
+* Ember services vs. global variables
+* Mocking services in tests
+
+<h3><a href="../13-ember-data/">Chapter 13</a></h3>
+
+* Ember Data models
+* Testing models
+* Loading models in routes
+* The Ember Data store
+* Working with adapters and serializers
+
+<h3><a href="../14-provider-components/">Chapter 14</a></h3>
+
+* Using Ember's built-in `<Input>` helper
```

Comment:
```suggestion
* Using Ember's built-in `<Input>` component
```

Reply:
looks like we had this in the provide component chapter as well. i just pushed up a fix for both instances